python/ray/train/tests/lightning_test_utils.py

```diff
@@ -22,24 +24,31 @@ def training_step(self, batch):
 
     def validation_step(self, val_batch, batch_idx):
         loss = self.forward(val_batch)
+        self.loss.append(loss)
         return {"val_loss": loss}
 
-    def validation_epoch_end(self, outputs) -> None:
-        avg_loss = torch.stack([x["val_loss"] for x in outputs]).mean()
+    def on_validation_epoch_end(self) -> None:
+        avg_loss = torch.stack(self.loss).mean()
         self.log("val_loss", avg_loss)
+        self.loss.clear()
```

Comment:
So, did `on_validation_epoch_end` exist in lightning 1.x? Or, did `validation_epoch_end` just get removed in 2.x? Wondering why we had to make this change.